.github/workflows/workflow-lint.yml

```
@@ -0,0 +1,41 @@
+name: Lint GitHub Actions workflow files
+
+on:
+  push:
+    branches:
+      - trunk
+      - '[0-9].[0-9]'
```

In the past when introducing new workflows, I've tried to use only future facing patterns starting with when the workflow was introduced. I don't feel strongly about not doing that here, but may be nice for consistency.
```suggestion
      - '6.[8-9]'
      - '[7-9].[0-9]'
```
This would require an update once we get to version `10.0`, but based on 3 releases per year, that's not for at least 7 years from now.

---

Makes sense!